src/features/sub-tasks-progress/BoardSettings/ColorSchemeSettings/SubTasksSettings.tsx

```
@@ -0,0 +1,106 @@
+        alignItems: 'center',
+      }}
+    >
+      {Object.entries(statuses).map(([statusId, status]) => {
+        const statusIdNumber = parseInt(statusId, 10);
+        if (Number.isNaN(statusIdNumber)) {
+          return null;
+        }
+        const projects = status.projects.join(', ');
+        return (
+          <>
+            <span data-testid="subtasks-settings-status-name"> {status.name}</span>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)